pystibmivb/service/STIBService.py

```diff
@@ -27,7 +27,7 @@ async def get_passages(self, stop_name, line_filters=None, max_passages=15, lang
         stop_infos = await self._shapefile_service.get_stop_infos(stop_name)
 
         if lang is None or lang == '':
-            lang = ('fr', 'en')
+            lang = ('fr', 'nl')
```

Comment:
Put ('fr','fr') as default (or even 'nl', 'nl' i don't care but it has to be consistent)

Reply:
I didn't understand your logic here.
We have LANG_MESSAGE = 1  by default
So when we have 
```
                    message = ""
                    try:
                        message = json_passage["message"][lang[LANG_MESSAGE]]
```
we have lang[1] ("en"), so I changed it to lang={"fr","nl"} but checking better, I see there's no way to change LANG_MESSAGE, get_passages() expects a tuple for lang and not a value for LANG_MESSAGE.
So how do you change the language for the messages?
Do you expect to do get_passages("Saint-Denis", lang={"fr","fr"})?
Why not having something like lang = "fr", and do a check if lang is in {"fr","nl"} ?

Reply:
Yeah it seems that messages can be in three languages (fr,nl,en) but stop names are only in two (fr,nl) 
The lang variable is a tuple containing the configuration for the lang for the stop_names (index=0) and the lang for the messages (index=1)

Do you expect to do get_passages("Saint-Denis", lang={"fr","fr"})? => yes

I haven't put a check to ensure that the lang was in 'fr','nl' or 'fr','nl','en' 

In future versions i plan on adding more input verfications and more exceptions everywhere but i wanted to keep it simple for me at the start.